statsd-exporter/include/mappings-gen2.yml

```diff
@@ -268,6 +268,90 @@ mappings:
+
+  # What is the name of the operator class used to run the task? Did the end-user subclass it?
+  # Which dbt command was used to run the task?
+  # What execution mode was used? What invocation mode was used?
+  - match: cosmos\.task\.operator_name\.([^.]+)\.is_subclass\.([^.]+)\.execution_mode\.([^.]+)\.invocation_mode\.([^.]+)\.dbt_command\.([^.]+)\.install_deps\.([^.]+)\.origin\.([^.]+)\.has_callback\.([^.]+)\.status\.([^.]+)\.counter$
+    match_type: regex
+    name: "cosmos_task_overview"
+    labels:
+      operator_name: "$1" # Example: "DbtRunLocalOperator", "DbtProducerWatcherOperator"
+      is_subclass: "$2" # True or False
+      execution_mode: "$3" # Example: "local", "virtualenv", "watcher"
+      invocation_mode: "$4" # Subprocess or DbtRunner
+      dbt_command: "$5" # Example: "run", "build", "test"
+      install_deps: "$6" # True or False
+      origin: "$7" # DbtTaskGroup, DbtDag or StandaloneTask
+      has_callback: "$8" # True or False
+      status: "$9" # success, failure
+
+  # Which database did the user run the transformations against?
+  # How did the user define the profile, via a ProfileMapping or YAML file?
+  # If Profile mapping, which class was used to define the profile?
+  - match: cosmos\.profile\.database\.([^.]+)\.profile_strategy\.([^.]+)\.profile_mapping_class\.([^.]+)\.counter$
+    match_type: regex
+    name: "cosmos_profile_overview"
+    labels:
+      database: "$1" # postgres, snowflake, databricks, bigquery, etc
+      profile_strategy: "$2" # yaml_file or mapping
+      profile_mapping_class: "$3" # None, SnowflakeEncryptedPrivateKeyPemProfileMapping, PostgresUserPasswordProfileMapping, DatabricksTokenProfileMapping, etc
+
+  # If using `DbtDag` or `DbtTaskGroup`, how did the user parse the dbt project? If using dbt_ls, was `dbt` in the same Python virtualenv as Airflow?
+  # Did the user specify a custom load converter?
+  # What were the source and node behaviors?
+  # How many dbt nodes were in the project?
+  # How many dbt nodes were selected by the user?
+  # How long did the parse operation take?
+  # How long did the filtering operation take?
+  - match: cosmos\.rendering\.used_automatic_load_mode\.([^.]+)\.actual_load_mode\.([^.]+)\.invocation_mode\.([^.]+)\.install_deps\.([^.]+)\.uses_node_converter\.([^.]+)\.test_behavior\.([^.]+)\.source_behavior\.([^.]+)\.total_dbt_models\.([^.]+)\.selected_dbt_models\.([^.]+)\.counter$
+    match_type: regex
+    name: "cosmos_rendering_overview"
+    labels:
+      used_automatic_load_mode: "$1" # True or False
+      actual_load_mode: "$2" # dbt_ls, dbt_ls_cache, dbt_ls_file, dbt_manifest, custom
+      invocation_mode: "$3" # subprocess or dbt_runner
+      install_deps: "$4" # True or False
+      uses_node_converter: "$5" # True or False
+      test_behavior: "$6" # after_each, after_all, none, build
+      source_behavior: "$7" # all, with_tests_or_freshness, none
+      total_dbt_models: "$8" # Total number of dbt models in the project
```

Comment:
Having numerical values as labels isn't really great, because you can't use them in calculations at all, and if they change they end up having an impact on cardinality.

statsd also has a gauge type, could you emit `total_dbt_models` and `selected_dbt_models` as gauges please?